app/src/Entity/Player.php

```diff
@@ -65,7 +65,7 @@ public function getShoots(): Collection
         return $this->shoots;
     }
 
-    public function getHash(): string 
+    public function getHash(): ?string 
```

Comment:
Sometimes the method returns `null`?

Reply:
Yes, when I create a game, I init a second player which does not have a hash yet.